Frontend/src/pages/registration_administration/components/RegistrationAdministrationList.jsx

```diff
@@ -8,8 +8,18 @@ import LoadingMessage from '../../../ui/loadingMessage'
 import styles from './list.module.scss'
 import RegistrationActions from './RegistrationActions'
 
+// Currently it is at the developer's discretion to make sure
+// an attendee is added to the right list.
+// One Solution would be to keep the registrations state as
+// the source of truth and partition as needed
 const reducer = (state, action) => {
   const { type, attendee } = action
+  // Make sure no one adds an attendee twice
+  if (
+    [...state.waiting, ...state.accepted, ...state.deleted].includes(attendee)
```

Comment:
Uh won't this block all the `remove-_` actions from removing?

Reply:
oops, it was too late yesterday